.travis.yml

```diff
@@ -16,15 +26,19 @@ matrix:
       env:
         - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
         - COVERAGE=true
-        - PHPUNIT_FLAGS="--coverage-clover build/coverage.xml"
+        - TEST_COMMAND="composer test-ci"
```

Comment:
Please check this again:

https://github.com/php-http/httplug/blob/master/.travis.yml#L26

Travis linter says multiline event params are invalid here